main.py

```
@@ -0,0 +1,62 @@
+    refresh_token: str
+    token_type: str
+    expires_in: int = 3600
+    sign_in_at: datetime
+
+class LoginResponse(BaseModel):
+    status: str
+    data: TokenData
+
+@app.post("/auth/token", response_model=LoginResponse)
+async def login_access_token(
+        token_in: LoginRequest,
```

Ruff (0.14.11)

Unused function argument: `token_in`

(ARG001)